app/app.go

```diff
@@ -490,7 +491,7 @@ func NewSommelierApp(
 	// Setup an upgrade handler if doing an upgrade module upgrade
 	app.UpgradeKeeper.SetUpgradeHandler(
 		upgradeName,
-		func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+		func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
```

Comment:
If we're basing this off of the Gaia upgrade, is there any necessity for this adjustment to the IBC keeper?

https://github.com/cosmos/gaia/blob/329796922bc8aedfcd5c81820357185857b23330/app/app.go#L570

Reply:
So gaia was upgrading from pre-1.0 IBC to IBC 2.0 and needed line 570 to deal with `https://github.com/cosmos/ibc-go/blob/main/CHANGELOG.md?plain=1#L258`

Reply:
👍